applications/FluidDynamicsApplication/custom_conditions/navier_stokes_wall_condition.h

```diff
@@ -522,6 +545,13 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) NavierStokesWallCondition : public
     ///@name Member Variables
     ///@{
 
+    Element* mpParentElement = NULL;
```

Comment:
in modern C++ this should be `nullptr`

Btw is it a good idea to save this?

Reply:
Since the parent finding mechanism is not overly expensive, the search is repeated instead of storing a pointer. Thank you very much for the hint.